frontend/src/app/commonComponents/Dropdown.jsx

```diff
@@ -2,13 +2,27 @@ import React from "react";
 import PropTypes from "prop-types";
 import { v4 as uuidv4 } from "uuid";
 
-const Dropdown = ({ options, label, name, onChange, selectedValue }) => {
+const Dropdown = ({
+  options,
+  label,
+  name,
+  onChange,
+  disabled,
+  defaultOption, // value of the default option
+  selectedValue,
+}) => {
   const optionsElements = options.map(({ value, label }) => (
     <option key={`dropdown-${uuidv4()}`} value={value}>
       {label}
     </option>
   ));
 
+  optionsElements.unshift(
+    <option key={`dropdown-${uuidv4()}`} value={null}>
```

Comment:
not sure if `null` is a valid value for `options`. Might need to be an empty string